README.md

```diff
@@ -3,7 +3,7 @@
 Generate an http4s service on the blaze backend with Circe.
 
 1. [Install sbt](http://www.scala-sbt.org/0.13/docs/Setup.html)
-2. sbt -sbt-version 0.13.13 new http4s/http4s.g8
+2. sbt -sbt-version 0.13.16 new http4s/http4s.g8
```

Comment:
Does this version of sbt bundle a g8 that supports the stable identifier?  That's what held us back before.  Is there any reason this command shouldn't reference sbt 1.0?